_includes/v23.1/sidebar-data/manage.json

```diff
@@ -144,6 +144,12 @@
                 "/${VERSION}/take-and-restore-encrypted-backups.html"
               ]
             },
+            {
+              "title": "Execution Locality Backups",
```

Comment:
I'm not in love with this page title or URL but maybe it's too late to change it. What about "Restrict backups by locality" or something like that?

Reply:
We have changed the title a bit, but this might evolve with some more feedback we're seeking from the field.